spaces/domains.json

```
@@ -1,5 +1,6 @@
 {
   "vote.cryptohobos.io": "cryptohobos.eth",
+  "vote.slimysnails.io": "slimysnails.eth"
```

A comma is missing at the end of your line.
```suggestion
  "vote.slimysnails.io": "slimysnails.eth",
```